### DISCUSSION TITLE
Keeping deployed services up during judging period

### DISCUSSION THREAD

**Discussion by cjrieth** [microsoft/AI-Chat-App-Hack#99] [Q&A] [answered]
Is there a good way to minimize the cost of our deployed services during the judging period?

I am using the basic SKU for Azure AI Search due to the size of my data, and as a result I am incurring some cost. This has been fine during development since I can easily take down resources when I  am done with them, but I am not sure how long I will need to have it up after I turn the project in.

Is there any way to know when our project will be in use and when it can be down? Or it is required that our service always be available until judging is complete.

Thanks!

**Comment by IsaacGemal** [comment]
Agreed. I'm in a similar predicament.
I'm going to take mine down after the showcase later today and maybe just upload another YouTube video showing what it can do as a substitute.

**Comment by MatheusHRV** [comment]
I think that we can schedule the deployment in Azure, we just need to know when it will be seen.

**Comment by pamelafox** [comment, marked as answer]
We're going to try to do all the judging this week, but if you need to take it down earlier, please record a video of you using your app demonstrating all functionality and link to that video from your entry, if you have not already. Thanks for asking!
